.drone.yml

```diff
@@ -0,0 +1,48 @@
+---
+kind: pipeline
+type: kubernetes
+name: default
+
+trigger:
+  branch:
+    - master
+
+steps:
+  - name: test
+    image: golang:1.13
+    commands:
+      - go vet ./...
+      - go test ./...
+
+  - name: publish_push
+    image: plugins/docker
+    settings:
+      registry: quay.io
+      repo: quay.io/mongodb/drone-bazelisk-ecr
+      tags:
+        - git-${DRONE_COMMIT_SHA:0:7}
+        - latest
+      username:
+        from_secret: docker_username
```

Comment:
Should we call these `quay_*` to avoid confusion with docker hub?

Reply:
Perhaps, but for now this matches how we have this configured for all of our other repos. So I'll leave that exercise for a future date if this ever needs to be deployed to docker hub (unlikely).